plat/common/aarch64/platform_helpers.S

```diff
@@ -55,31 +55,31 @@ func plat_report_exception
 endfunc plat_report_exception
 
 	/* -----------------------------------------------------
-	 * Placeholder function which should be redefined by
-	 * each platform.
+	 * int plat_crash_console_init(void)
+	 * Use normal console by default. Switch it to crash
+	 * mode so serial consoles become active again.
 	 * -----------------------------------------------------
 	 */
 func plat_crash_console_init
-	mov	x0, #0
-	ret
+	b	console_start_crash
 endfunc plat_crash_console_init
 
 	/* -----------------------------------------------------
-	 * Placeholder function which should be redefined by
-	 * each platform.
+	 * void plat_crash_console_putc(int character)
+	 * Output through the normal console by default.
 	 * -----------------------------------------------------
 	 */
 func plat_crash_console_putc
-	ret
+	b	console_putc
```

Comment:
Actually, it's not clear that there are any C consoles out there, so this could be a sensible change. However, it should probably be in a separate patch is it doesn't seem part of the general API change.